### DISCUSSION TITLE
part-ii/2021-71/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#323] [Question comments]
Part II 2021 Galois Theory: Paper 4 , Section II, 18I 

https://questions.tripos.org/part-ii/2021-71/

**Comment by DannyDuberstein** [comment]
(b) orbit size
(c) write down the minpoly of z/L^G as in proof of artin, coefficients of a min poly generate the field

**Reply by artie2000** [reply]
What do you mean for part (c)? What was your element $w$?

**Reply by DannyDuberstein** [reply]
Same as yours, the min poly of z over L^G is $(X-\zeta z)(X-\zeta^2 z)...(Z-\zeta^{-1}z)...(X-\zeta^n z) = (X^n - z^n)(X^n - z^{-n}) = X^{2n} - (z^n + z^{-n})X^n + 1$

**Reply by artie2000** [reply]
Ah, that's a nice solution

**Comment by artie2000** [comment]
Can take $w=z^n+\frac{1}{z^n}$.
